src/partials/40_section-sale.html

```diff
@@ -0,0 +1,46 @@
+<div class="section--sale">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-6">
+        <div class="image-box large">
+          <div class="image" style="background-image: url(images/sale/sale-image1.jpg)">
```

Comment:
Generalnie nie ma większych zastrzeżeń, więc przyczepię się tylko do mały rzeczy ;) Poniżej w 19 lini masz diva z tą samą klasą oraz stylem background-image w inline, tylko tam zastosowałaś dodatkowe wcięcia - żeby było idealnie proponuję zostać przy jednym sposobie. :)

Reply:
to wynika z ustawień maksymalnej dopuszczalnej długości linijki kodu ;)